lua/ui/lobby/lobbyOptions.lua

```diff
@@ -193,6 +194,30 @@ teamOptions =
             }
         }
     },
+
+    {
+        default = 2,
+        label = "<LOC lobui_resource_sharing_label>Share excess resources",
+        help = "<LOC lobui_resource_sharing_help>Allows you to enforce whether the sharing of excess resources with allies is enabled, disabled or toggleable.",
```

Comment:
```suggestion
        help = "<LOC lobui_resource_sharing_help>Set whether distribution of excess resources to allies is always enabled, always disabled, or toggleable by players.",
```
1. Match the perspective/language of other lobby options and the tooltip for the resource share button.
2. Since non-host players can't see option descriptions in the lobby, be more specific in the main option's description.

Reply:
Applied in https://github.com/FAForever/fa/pull/6938/commits/a8e59e6539e382a3dccef354b4d0b2cba4e80029.